src/eq_schema/Questionnaire.js

```diff
@@ -13,13 +14,29 @@ class Questionnaire {
     this.groups = this.buildGroups(authorJson.sections);
     this.theme = authorJson.theme;
     this.legal_basis = authorJson.legalBasis;
+    this.navigation = this.buildNavigation(
+      authorJson.navigation,
+      authorJson.sections
+    );
   }
 
   buildGroups(sections) {
     const groups = sections.map(section => new Group(section));
     last(groups).blocks.push(new Summary());
     return groups;
   }
+
+  buildNavigation(visible, sections) {
+    return {
+      visible,
+      sections: sections.map(section => {
+        return {
+          title: getInnerHTML(section.title),
+          group_order: [`group-${section.id}`]
```

Comment:
out of curiosity, what does survey runner do with this?

Reply:
I'm not entirely sure why it needs this, but on first glance it uses the `group_order` for checking if the group is to be skipped or not, and for determining which section should be highlighted. Perhaps @ajmaddaford can provide some additional context?